lms/templates/navigation.html

```diff
@@ -100,6 +100,11 @@ <h2 class="course-header"><span class="provider">${course.display_org_with_defau
 
       <%include file="user_dropdown.html"/>
 
+      <a href="${get_online_help_info(online_help_token)['doc_url']}" 
```

Comment:
I removed the title. That was wordy. I had just copied it from the equivalent link in Studio.